app/views/help/cookie_settings.html.haml

```diff
@@ -0,0 +1,66 @@
+- if params[:'analytical-cookies'] == 'off'
+  .govuk-notification-banner
+    .govuk-notification-banner__header{ "role": "region", "aria-labelledby": "govuk-notification-banner-title", "data-module": "govuk-notification-banner" }
+      %h2.govuk-notification-banner__title#govuk-notification-banner-title
+        = t('cookie_settings.notification_banner.heading')
+    .govuk-notification-banner__content
+      %p.govuk-notification-banner__heading
+        = t('cookie_settings.notification_banner.turned_off')
+
+- if params[:'analytical-cookies'] == 'on'
+  .govuk-notification-banner
+    .govuk-notification-banner__header
+      %h2.govuk-notification-banner__title#govuk-notification-banner-title
+        = t('cookie_settings.notification_banner.heading')
+    .govuk-notification-banner__content
+      %p.govuk-notification-banner__heading
+        = t('cookie_settings.notification_banner.turned_on')
+
+= govuk_page_title(t('cookie_settings.page_title'))
+
+%p.govuk-body= t('cookie_settings.section_intro.paragraph_one')
+
+%p.govuk-body= t('cookie_settings.section_intro.paragraph_two')
+
+%h2.govuk-heading-l= t('cookie_settings.section_necessary.heading')
+
+%p.govuk-body= t('cookie_settings.section_necessary.paragraph_one')
+
+%ul.govuk-list.govuk-list--bullet
+  %li= t('cookie_settings.section_necessary.list_item_one')
+  %li= t('cookie_settings.section_necessary.list_item_two')
+
+%p.govuk-body= t('cookie_settings.section_necessary.paragraph_two')
+
+%p.govuk-body
+  = link_to t('cookie_settings.section_necessary.link_to_cookie_page'), cookies_path, class: 'govuk-link'
+
+.app-no-js-only
+  %h2.govuk-heading-l= t('cookie_settings.section_settings.heading_no_js')
+
+  %p.govuk-body= t('cookie_settings.section_settings.paragraph_one_no_js')
+
+  %p.govuk-body= t('cookie_settings.section_settings.paragraph_two_no_js')
+
+%form#cookie-form.app-js-only{method: "get", data: {module: "cookie-settings"}}
+
+  %h2.govuk-heading-l= t('cookie_settings.section_settings.heading')
+
+  %p.govuk-body= t('cookie_settings.section_settings.paragraph_one')
+
+  %fieldset.govuk-fieldset{ :aria-describedby => "cookie-hint-1" }
+    %legend.govuk-fieldset__legend.govuk-fieldset__legend--m
+      = t('cookie_settings.form.heading')
+      %span#cookie-hint-1.govuk-hint
+        %p= t('cookie_settings.form.hint')
+      .govuk-radios
+        .govuk-radios__item
+          %input#ga_on.govuk-radios__input{ :type => "radio", :name => "analytical-cookies", :value => "on", :type => "radio" }
```

Comment:
Standard: use the new json style hash syntax `type: 'radio'` instead of the rocket hash syntax `:type => 'radio'`

Replace these instances throughout your PR.

Reply:
Done (I think)